nornir/core/deserializer/configuration.py

```diff
@@ -185,6 +187,6 @@ def _resolve_import_from_string(import_path: Any) -> Optional[Callable[..., Any]
         module_name, obj_name = import_path.rsplit(".", 1)
         module = importlib.import_module(module_name)
         return getattr(module, obj_name)
-    except Exception as e:
-        logger.error(f"failed to load import_path '{import_path}'\n{e}")
+    except Exception:
+        logger.error("Failed to import %r", import_path, exc_info=True)
```

Comment:
I'd like to use f-strings everywhere as they are easier to read and execution time they are identical